packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/edit-connector/edit-connector.html

```diff
@@ -0,0 +1,75 @@
+<oui-back-button
+    data-previous-page="Connectors"
+    data-on-click="$ctrl.goBack()"
+></oui-back-button>
+
+<!-- error messages -->
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+<div class="d-flex justify-content-between align-items-center mb-4">
+    <div>
+        <h2 ng-bind="$ctrl.availableConnector.name" class="mb-0"></h2>
+        <span class="oui-paragraph">
+            Version: {{$ctrl.availableConnector.version}} | Author:
+            {{$ctrl.availableConnector.author}} |
+            <a
+                ng-href="{{$ctrl.availableConnector.documentationUrl}}"
+                target="_blank"
+                rel="noopener"
+                >docs</a
+            >
+        </span>
+    </div>
+    <oui-button variant="primary" size="l" on-click="$ctrl.updateConnector()">
+        <span data-translate="pci_databases_connectors_edit_button"></span>
+    </oui-button>
+</div>
+
+<oui-tabs>
+    <oui-tabs-item
+        ng-repeat="group in $ctrl.availableConnector.configuration.getGroups() track by $index"
+        heading="{{group.name}}"
+        subheading="{{group.required ? 'required' : ''}}"
+        variant="promotion"
+    >
+        <div class="row">
+            <div class="col-md-6">
+                <oui-message type="info" ng-if="group.name === 'Transforms'">
```

Comment:
Nitpick: can you move "group.name" values to a ENUM into constant file instead of hardcoding it here

Reply:
Done here: https://github.com/ovh/manager/pull/6579/commits/096202bbb90f6a96e19061785069f7c99834ecc4